x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/profile/ProfileService.java

```diff
@@ -305,6 +329,52 @@ private void getVersionedDocument(String uid, ActionListener<VersionedDocument>
         });
     }
 
+    private void getVersionedDocuments(
+        Collection<String> uids,
+        ActionListener<Tuple<Map<String, VersionedDocument>, List<String>>> listener
```

Comment:
We can simplify this to
```suggestion
        ActionListener<Tuple<List<ProfileDocument>, List<String>>> listener
```

because:
1. `ProfieDocument` already has `profileUid` so no need for the map key
2. We don't need version control info in this use case so no need to wrap it within a VersionedDocument.

Reply:
Good tip, thank you!